sample-config.toml

```diff
@@ -11,8 +11,12 @@ QNoTLS=true #Whether to start TLS corresponding to qitmeer
 QTLSSkipVerify=true
 WalletPass="public" #Wallet encryption code default public
 
-MinTxFee=200000   # The minimum transaction fee in QIT/KB default 200000 (aka. 0.002 MEER/KB)
-Confirmations=10   # Number of block confirmations
+# The minimum transaction fee in QIT/KB default 200000 (aka. 0.002 MEER/KB)
+MinTxFee=200000
+
+# Number of block confirmations. Miners are recommended to set above 720,
+# and ordinary users are recommended to set above 10.
+Confirmations=10
```

Comment:
what if miners are also ordinary users, or miners need to spend their change of coinbase. 

for instance, saying the Confirmations=800  the mining pool spend 100 to reward miners and leave 20 change to himself.
If he wants to spend his this change 20, do he need to wait 800 confirmations?